PRs fail because of the new version of protobuf being pulled

/kind bug

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
Raise a PR. python package build fails because in some envs the version of protobuf pulled is incompatible.

```
Collecting protobuf>=3.12.0
  Downloading protobuf-4.21.0-cp37-abi3-manylinux2014_x86_64.whl (407 kB)
```

```
E   TypeError: Descriptors cannot not be created directly.
E   If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
E   If you cannot immediately regenerate your protos, some other possible workarounds are:
E    1. Downgrade the protobuf package to 3.20.x or lower.
E    2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
E   
E   More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

**What did you expect to happen:**
PR to pass


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Istio Version:
- Knative Version:
- KFServing Version:
- Kubeflow version:
- Kfdef:[k8s_istio/istio_dex/gcp_basic_auth/gcp_iap/aws/aws_cognito/ibm]
- Minikube version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

--- a/python/kserve/requirements.txt
+++ b/python/kserve/requirements.txt
@@ -26,3 +26,4 @@ cffi==1.15.0
 idna==3.3
 certifi==2021.10.8
 tritonclient==2.18.0
+protobuf~=3.19.0

